types/mercadopago/tslint.json

```diff
@@ -0,0 +1,6 @@
+{ 
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "npm-naming": [true,{"mode":"code","errors":[["NeedsExportEquals",false]]}]
```

Comment:
Remove the rules here and fix the errors.. you cannot disable any rules for new package.

Reply:
Doing so and running `npm run test mercadopago` would give me this error:
```
The declaration doesn't match the JavaScript module 'mercadopago'. Reason:
The declaration should use 'export =' syntax because the JavaScript source uses 'module.exports =' syntax and 'module.exports' can be called or constructed.

To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
If you won't fix this error now or you think this error is wrong,
you can disable this check by adding the following options to your project's tslint.json file under "rules":

    "npm-naming": [true,{"mode":"code","errors":[["NeedsExportEquals",false]]}]
 See: https://github.com/Microsoft/dtslint/blob/master/docs/npm-naming.md
```
I have tryed using a class and all properties being static, but the linter will tell me to use a namespace or plain object. And I also think this makes more sense. So using namespace the only way I can make this work is by doing:
```
declare namespace MercadoPago {
  const configure: MercadoPagoConfig['configure'];
  const configurations: MercadoPagoConfig;
  ...
}

// tslint:disable-next-line export-just-namespace
export = MercadoPago;
```
Is this correct? (I am commiting this change anyway and I'll fix later if it is wrong.) If not what is the right one? I have no ideia how this would be with plain objects